### ISSUE TITLE
install node-red-contrib-speakerpi error

### ISSUE THREAD

**Issue by JanOosthuizen** [O-Hahn/node-red-contrib-speakerpi#10]
Regarding : node-red-contrib-speakerpi
I get errors when I try to install node-red-contrib-speakerpi.

When running command: "npm install node-red-contrib-speakerpi" I get the following message -

**Error message:**
gyp: name 'mpg123_cpu' is not defined while evaluating condition 'mpg123_cpu=="arm_nofpu"' in deps/mpg123/mpg123.gyp while loading dependencies of binding.gyp while trying to load binding.gyp
gyp ERR! configure error
gyp ERR! stack Error: `gyp` failed with exit code: 1
.....
..

**(See link for full message and[ log file](url))**
h @ttps://drive.google.com/drive/folders/1DFDc43DIVX07Mel_z_CYpD67paaFsWNo?usp=sharing

**Hardware & Software used:**
Raspberry Pi 4 + Running from External SSD
OS-release    Debian11 (Bullseye) 64Bit
Node-Red v2.2.6
node v14.19.1
Node.js v12.22.5
Linux V5.15
NPM 6.14.16
node-gyp (Looks like node-gyp is NOT installed)

**I tried the following with no success-**
1) sudo npm install node-red-contrib-speakerpi
2) sudo npm install -g --unsafe-perm node-red-contrib-speakerpi     

 Will appreciate any advice or help

**Comment by juggledad** [comment]
Did you follow the directions in the readme and do the pre-req's? (like installing  libasound2-dev)

**Comment by JanOosthuizen** [comment]
Yes,  I did . I made double sure  installing libasound2-dev. (I have used it before on my old Pi3 32Bit with no problems)
The new installation is on a Pi4 with external SSD and 64Bit Bullseye OS)

**Comment by juggledad** [comment]
This could be a 'Buster' vrs 'Bullseye' or a 32 bit vrs 64 bit issue. You coud try flashing a couple SD cards and testing them to check if that is the basis of the issue.

Note: just to be sure, youjust wrote:
`The new installation is on a Pi4 with external SSD and 64Bit Buster OS)`
but that conflicts with your original post - if your are running 'Bullseye' you should edit your last post.
